src/Xenial.Tasty.Core/Reporters/Reporters.ColorScheme.cs

```diff
@@ -50,6 +50,6 @@ public ColorSchemeLegacy()
             IgnoredIcon = "‼";
         }
 
-        public static new ColorScheme Default = new ColorSchemeLegacy();
+        public new static ColorScheme Default => new ColorSchemeLegacy();
```

Comment:
That would break the configurability cause this will create a new ColorScheme every time you access the `Default` property. So this should be a readonly getter:

```cs
public new static ColorScheme Default { get; } new ColorSchemeLegacy();
```

Reply:
Corrected in 5fbda570b0b835965f7aa1dd3a7070bd5857e8d8